### DISCUSSION TITLE
How are the antennas of all the modules combined?

### DISCUSSION THREAD

**Discussion by jaytxrx** [pascallanger/DIY-Multiprotocol-TX-Module#800] [Q&A] [answered]
I understand that in the multiprotocol module there are four separate boards for each RF protocol.

In the Chinese 4in1 module there is only one antenna for all the 4 RF modules. Any idea how are they combined?

Is there a special antenna combining circuit? I am asking because if one Modul is active and others deactivated, the RF power will flow into the others and damage. Or is it that since there is an antenna always connected, there won't be RF power reflections and hence no damage ?

**Comment by pascallanger** [comment, marked as answer]
There is a RF switch to select the RF signal from one of the four components.
